Interview-English (#201)
techwriter-pl/techwriter-pl.github.io: blog/kilka-pytan-do-czesc-31-English/index.md

---
title: 'Interview with Mateusz Barszcz'
authors: niewiarowski
date: '2025-03-31'
tags:
  - 'warsztat'
  - 'wywiady'
  - 'kariera'
  - 'praca'
  - 'articles-in-english'
coverImage: 'upload_0007.png'
---

Our guest today is [Mateusz Barszcz](https://www.linkedin.com/in/mattbarszcz/),
a Chief Software Technical Writer at
[Hitachi Energy](https://www.linkedin.com/company/hitachienergy/) in Krakow,
Poland. Let's hear his story, this time in English. For the Polish version,
[click here](https://techwriter.pl/kilka-pytan-do-czesc-31). In this interview,

> you will learn about his extensive experience in technical writing and the

Reviewer: ```suggestion
you will learn about his extensive experience in technical writing and how
```

> respects in which this domain resembles competitive sports. Mateusz shares his

Reviewer: ```suggestion
how technical writing is like competitive sports. Mateusz shares his
```

> invaluable observations and thoughts, which will be helpful for those

Reviewer: ```suggestion
invaluable observations and insights, which will be helpful for those
```

> considering a step toward technical communication as well as anyone interested

Reviewer: ```suggestion
who want to get started with technical communication, nut also for anyone else interested
```

in it.

<!--truncate-->

![A technical writer sitting in a car](images/upload_0099.png)

## Introduction

### Hi, thank you for agreeing to this interview. To start, could you tell us something about yourself?

My name is Mateusz Barszcz, and I work at Hitachi Energy, where I create
technical documentation. I have been a member of the Technical Writers team
since late 2022, when I joined as a Senior Technical Writer. Initially, I worked
on a smaller project called
[Network Manager Wide Area Monitoring System (WAMS)](https://www.hitachienergy.com/products-and-solutions/grid-and-generation-management-network-manager/network-manager-wide-area-monitoring-system-wams),
a backend solution for monitoring the state of the power grid.

Currently, I’m involved in a much larger project with five development teams
engaged in the process, serving as the lead technical writer. In that role, I
create the documentation plan and ensure we have all the essential information

> needed to proceed with writing. As a writer, I describe user documentation for

Reviewer: ```suggestion
needed to proceed with writing. As a writer, I write user documentation for
```

device in a police car similar to the one Mateusz dealt with.**
[Link to the original photo](https://www.flickr.com/photos/peigov/31808990565/in/photostream/).
Author:
[Government of Prince Edward Island](https://www.flickr.com/photos/peigov/).

Then, I wrote training materials for e-learning, which involved managing content
in a different format. I created materials for self-paced learning and
instructor-led training about the radio infrastructure I learned initially. I
was managing well-known content in a different format. As an instructional
designer, I designed trainings for enablement and business topics. The scope of
content I operated on changed dynamically.

Then, I accepted an offer to work as a contractor at Google. I joined a small
team writing technical content and creating online training for internal project
management products. It was a new and completely unknown area for me. I needed
to learn many tools designed for employees. It was a dynamic role which I spent
over six years in. As the product portfolio developed, my responsibilities
expanded. Ultimately, the contracted project ended, and I had to look for
something new.

## Career at Hitachi Energy

> ### Let’s return to your current role at Hitachi Energy, which you mentioned at the beginning. How is your work organized?

Reviewer: ```suggestion
### Let’s return to your current role at Hitachi Energy. How is your work organized?
```

### What are the biggest challenges you face in your work?

Sometimes, you must convince someone that the path you propose is correct and
make it clear to someone that they might not be right themselves. It is not
pleasant to impose your opinion on someone. Still, there are specific standards
to maintain, and it is necessary. Occasionally, you also have to remind
engineers about delivering input. Everyone is busy, and in such cases,
one-on-one meetings work well.

### What do you like most about your job?

I like our Technical Writing team. In the beginning, there were only a few of
us, and now the team is growing rapidly and is very diverse. I will compliment
our team leader, Piotrek Sroka. I really like his approach to team management.
Another thing is the technology in which the project is created and the openness
of the development team to new techniques. This team proposes interesting
solutions that can be built on Markdown.

## Interests

### What are your interests outside of documentation and technology?

My passions are traveling, reading (sci-fi especially), automotive, and cooking.
Recipes also relate to instructions.

> ### Speaking of cooking, when you buy new household appliances, such as a kitchen robot, do you go with the flow and spontaneously start using it, or do you read the entire manual from cover to cover?

Reviewer: ```suggestion
### Speaking of cooking, when you buy new household appliances, such as a food processor, do you go with the flow and spontaneously start using it, or do you read the entire manual from cover to cover?
```
